### PR TITLE
[PIL-1021][AV] - Copy changes (Error screens across UPE/NFM Registration)

### PR DESCRIPTION
**Pull request by aashik-varahan**

### CODE REVIEW

File: app/views/fmview/NfmRegisteredAddressView.scala.html [hunk 1]
```diff
@@ -47,6 +48,8 @@
 
         @heading(messages("nfmRegisteredAddress.heading", UserName), "govuk-heading-l long-word")
 
+        @warningText(Html(messages("nfmRegisteredAddress.warningText")))
```
Comment (TevynAllen) [inline app/views/fmview/NfmRegisteredAddressView.scala.html:51]: Do you think it'll be good to have a spec for this?

File: app/views/registrationview/UpeRegisteredAddressView.scala.html [hunk 2]
```diff
@@ -48,6 +49,8 @@
 
         @heading(messages("upeRegisteredAddress.heading", UserName), "govuk-heading-l long-word")
 
+        @warningText(Html(messages("upeRegisteredAddress.warningText")))
```
Comment (TevynAllen) [inline app/views/registrationview/UpeRegisteredAddressView.scala.html:52]: ditto?

File: app/views/fmview/NfmRegisteredAddressView.scala.html (hunk 1 above)
Comment (aashik-varahan) [reply app/views/fmview/NfmRegisteredAddressView.scala.html:51]: We currently don't have test specs for the View classes. Reg and Sub journey views are covered in ATs.

Comment (AKKUM) [reply app/views/fmview/NfmRegisteredAddressView.scala.html:51]: @aashik-varahan could you please create a ticket for refactor of views for reg and subs. That can be probably looked at later.

File: app/views/registrationview/UpeRegisteredAddressView.scala.html (hunk 2 above)
Comment (AKKUM) [reply app/views/registrationview/UpeRegisteredAddressView.scala.html:52]: @aashik-varahan will create refactor ticket for the same

**Review by AKKUM — DISMISSED** [review]

**Review by javed-iqbal1 — DISMISSED** [review]

**Review by jassalrichy — APPROVED** [review]

**Pull request merged**